packages/osd-pm/src/README.md

```diff
@@ -1,269 +0,0 @@
-# `@osd/pm` — The OpenSearch Dashboards project management tool
```

Comment:
did we just have a duplicate of this file?

Reply:
Yes, this is dupliate. That is why I removed this file: packages/osd-pm/src/README.md